app/controllers/public/sessions.js

```diff
@@ -6,19 +6,19 @@ const { Controller, computed } = Ember;
 export default Controller.extend({
   sessionsByTracks: computed('model.sessions.[]', 'selectedTrackId', function() {
```

Comment:
This entire computed property to be removed and dynamic segments to be used instead of it to load sessions based on tracks.